### ISSUE TITLE
Download package failure for update_ubuntu_kernel.sh

### ISSUE THREAD

**Issue by jgeiger** [mtompkins/linux-kernel-utilities#11]
Every time I run the script now, I get this error when trying to update a kernel. It seems to have started just after 4.10.15 became available as I was able to install 4.10.14.

The script is able to find all available kernels, just fails at the download step. This also fails for the 4.11 kernel as well. I didn't try any of the older versions, but I assume it's similar.

```
Select your desired kernel: 3

Do you want the lowlatency kernel? (y/[n]):

[+] Processing selection
 \_ Determining CPU type: amd64
 \_ Locating source of v4.10.15 generic kernel packages.
```

```
There was an error @ Line: Download package failure.
There was a problem downloading the packages. Exiting.
Exiting with status code 127  
```

**Comment by mtompkins** [comment]
Yep, the Ubuntu packages have been malformed for a while (last 2 or 3 releases). If you download the files directly from the site you'll have the same failure trying to install with `dpkg`. Your options are to either wait on Canonical to fix their build system or to compile directly with `./compile_linux_kernel.sh`

FYI: You should do a `sudo apt-get install -f` to clean up the failure.

**Comment by mtompkins** [comment]
Tested again today. Confirmed the packages are now good yet the updater is having a problem. Will revert. Reopening ticket.

**Comment by mtompkins** [comment]
Merged with #10
